docs/user-guide/sso.md

```diff
@@ -104,7 +104,7 @@ GPUSTACK_SAML_SECURITY="{}"
 GPUSTACK_EXTERNAL_AUTH_NAME="emailaddress"
 GPUSTACK_EXTERNAL_AUTH_FULL_NAME="name"
 GPUSTACK_EXTERNAL_AUTH_AVATAR_URL="picture"
-GPUSTACK_EXTERNAL_AUTH_DEFAULT_ACTIVE="True"
+GPUSTACK_EXTERNAL_AUTH_DEFAULT_INACTIVE="True"
```

Comment:
Maybe "true" is better, given that most examples in https://docs.gpustack.ai/latest/cli-reference/start/#config-file use the lower case.